ironfish-cli/src/commands/peers/list.ts

```diff
@@ -10,6 +10,8 @@ import blessed from 'blessed'
 
 type GetPeerResponsePeer = GetPeersResponse['peers'][0]
 
+const StateColumnHeader = 'STATE'
```

Comment:
I think all the other files usually specify these as upper camel case with underscores, if you really want to pull this out.

```suggestion
const STATE_COLUMN_HEADER = 'STATE'
```

Reply:
I'll change that, main reason I pulled it out was just to make sure if we rename it in one place it gets updated in the other